apps/customer-portal/webapp/.env.example

```
@@ -12,3 +12,11 @@ CUSTOMER_PORTAL_THEME=oxygen_ui_theme
 
 # Logging Configuration (Choose one: DEBUG, INFO, WARN, ERROR)
 CUSTOMER_PORTAL_LOG_LEVEL=log_level
+
+# Global Notification Banner Configuration
+CUSTOMER_PORTAL_MAINTENANCE_BANNER_VISIBLE=false
+CUSTOMER_PORTAL_MAINTENANCE_BANNER_SEVERITY=info
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CUSTOMER_PORTAL_MAINTENANCE_BANNER_SEVERITY key should go before the CUSTOMER_PORTAL_MAINTENANCE_BANNER_VISIBLE key

(UnorderedKey)